src/chapters/local-dev/vscode-python.rst

```diff
@@ -152,7 +152,7 @@ Try It!
    Copy your code from there and paste it into VS Code. Some nice options
    include:
 
-   a. Your *Choose Your Own Adventure* activity.
+   a. Your *Choose Your Own Adventure*(There is no link for this, should we choose another activity) activity.
```

Comment:
The CYOA activity is a PDF handout the teachers receive during the summer training.  If assigned, the kids would do this activity right after learning about conditionals (chapter 5).

There is no link to the CYOA file (yet), since we haven't added these teacher resources to the book.  For now, let's throw in some quick context here.  Maybe along the likes of "...that your teacher assigned..."

Also, your (comment) throws a sphinx build error, so we'll need to check the syntax after you make the change.

Reply:
Done, I'm not getting a build error but let me know if you still do.